interface/main/messages/css/reminder_style.css

```diff
@@ -316,6 +316,8 @@ input[type="radio"] {
 .scheduled {
     background-color: white;
     color: black;
+    border-color: #333 !important;
+    padding: 5px !important;
```

Comment:
If possible, In here maybe we can use the Bootstrap built-class, e.g. `border border-secondary p-1`

Reply:
Are those classes available in 5.0.2?  This is a patch for 5.0.2.